anchor-reference-server/src/main/java/org/stellar/anchor/reference/event/KafkaListener.java

```diff
@@ -87,4 +96,50 @@ public void destroy() {
     consumer.close();
     stop();
   }
+
+  @Override
+  public int compareTo(@NotNull HealthCheckable other) {
+    return other.getName().compareTo(other.getName());
+  }
+
+  @Override
+  public String getName() {
+    return "kafka_listener";
+  }
+
+  @Override
+  public List<String> getTags() {
+    return List.of("all", "kafka");
+  }
+
+  @Override
+  public HealthCheckResult check() {
```

Comment:
should we test if the service is able to connect to kafka as well? ex: call consumer.listTopics() just to check the connection.

https://stackoverflow.com/questions/41016075/check-if-system-is-connected-to-kafka

Reply:
This is great suggestion. The check is added.